src/main/java/br/com/cwidevs/controller/IndexController.java

```diff
@@ -0,0 +1,17 @@
+package br.com.cwidevs.controller;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+/**
+ *
+ * @author pedro.rohr
+ */
+@Controller
+public class IndexController {
+
+    @RequestMapping({"/dashboard", "/jogadores", "/jogador", "/jogador/*", "/goleadores", "/partidas"})
```

Comment:
This is a code smell, that grows as the API endpoints evolves, needing always mapping new endpoints here.